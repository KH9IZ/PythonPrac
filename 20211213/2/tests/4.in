[82, 93, 46, 21, 85, 38, 47, 38, 85, 70, 93, 61, 30, 49, 0, 95, 16, 64, 42, 42, 35, 79, 6, 24, 46, 60, 10, 28, 51, 2, 77, 5, 44, 54, 69, 52, 18, 37, 19, 71, 55, 69, 40, 95, 49, 16, 95, 99]