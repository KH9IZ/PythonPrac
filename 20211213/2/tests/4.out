[0, 2, 5, 6, 10, 16, 16, 18, 19, 21, 24, 28, 30, 35, 37, 38, 38, 40, 42, 42, 44, 46, 46, 47, 49, 49, 51, 52, 54, 55, 60, 61, 64, 69, 69, 70, 71, 77, 79, 82, 85, 85, 93, 93, 95, 95, 95, 99]